CMakeLists.txt

```diff
@@ -8,11 +8,10 @@ set(CMAKE_CXX_STANDARD 17)
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
 project(IOOMEZarrNGFF CXX C ASM_NASM ASM)
-
 set(IOOMEZarrNGFF_LIBRARIES IOOMEZarrNGFF)
 
 if(NOT ITK_SOURCE_DIR)
-  find_package(ITK 5.0 REQUIRED)
+  find_package(ITK 5.4 REQUIRED)
```

Comment:
The upgrade of ITK seems unrelated to/independent of the rest of the commit ("Factor out itkOMEZarrNGFFCommon.h"). If so, preferably make it a separate commit.